# CONSUMER INSIGHTS ANALYSIS

**by Jayesh Suryawanshi**  

completed as part of Codebasics Project Challenge

**Domain:** F & B   **Function:** Marketing  
  

**About the company:**

CodeX is a German beverage company that is aiming to make its mark in the Indian market. A few months ago, they launched their energy drink in 10 cities of India.

Their Marketing team is responsible for increasing brand awareness, market share, and product development. They conducted a survey in those 10 cities and received results from 10k respondents. 


**Business Task:**  

To convert survey results to meaningful insights which the team can use to drive actions.  

**Business Objective:**  

To provide insights to the CodeX Marketing Team  

**Key Stakeholders:**  

Chief Marketing Officer  

**Information on the Data Source:**  
The data source consists of three excel files called **fact_survey_responses** which consists of the responses to various questions regarding reasons, perceptions, preferences, choices, expectations, etc., **dim_respondents** which consists of information regarding various respondents such as name, age, gender, etc. and **dim_cities** which consists of information about city names, tiers, etc.

**Tools Used:** PostgreSQL (Data Analysis), Tableau (Data Visualization)

In [1]:
%load_ext sql

In [2]:
import os

In [3]:
host = "localhost"
database = "Codebasics_Challenge_Project"
user = 'user_name'
password = 'password'

In [4]:
connection_string = f"postgresql://{user}:{password}@{host}/{database}"

In [5]:
%sql $connection_string

In [6]:
%%sql
select * from fact_survey_responses limit 10;


 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
10 rows affected.


[(103001, 120031, '2-3 times a week', 'To stay awake during work/study', 'Increased energy and focus', 'Yes', 'Neutral', 'Not sure', 'No', 5, 'Unfamiliar with the brand', 'Blue Bull', 'Availability', 'Reduced sugar content', 'Guarana', 'No', 'Yes', 'TV commercials', 'Compact and portable cans', 'Yes', '50-99', 'Supermarkets', 'Studying/working late'),
 (103005, 120035, 'Daily', 'To stay awake during work/study', 'Increased energy and focus', 'Yes', 'Neutral', 'Effective', 'Yes', 5, 'Other', 'Sky 9', 'Other', 'More natural ingredients', 'Caffeine', 'Yes', 'Yes', 'Online ads', 'Compact and portable cans', 'Yes', '100-150', 'Online retailers', 'Studying/working late'),
 (103006, 120036, 'Rarely', 'For mental alertness', 'To combat fatigue', 'Yes', 'Negative', 'Not sure', 'No', 5, 'Health concerns', 'Cola-Coka', 'Availability', 'Wider range of flavors', 'Vitamins', 'Yes', 'Yes', 'TV commercials', 'Other', 'Yes', 'Above 150', 'Gyms and fitness centers', 'Social outings/parties'),
 (103010, 120040, 'Once a week', 'For mental alertness', 'To combat fatigue', 'Yes', 'Neutral', 'Healthy', 'No', 4, 'Not interested in energy drinks', 'Sky 9', 'Effectiveness', 'Other', 'Caffeine', 'No', 'Yes', 'Other', 'Other', 'Not Sure', '50-99', 'Gyms and fitness centers', 'Social outings/parties'),
 (103012, 120042, 'Rarely', 'To stay awake during work/study', 'To combat fatigue', 'Yes', 'Neutral', 'Dangerous', 'No', 2, 'Not available locally', 'Others', 'Effectiveness', 'Reduced sugar content', 'Caffeine', 'Yes', 'Yes', 'Online ads', 'Other', 'Yes', '100-150', 'Online retailers', 'Social outings/parties'),
 (103017, 120047, 'Rarely', 'Before exercise', 'To boost performance', 'Yes', 'Neutral', 'Effective', 'No', 4, 'Not interested in energy drinks', 'CodeX', 'Taste/flavor preference', 'Wider range of flavors', 'Sugar', 'No', 'Yes', 'Print media', 'Innovative bottle design', 'Not Sure', '50-99', 'Supermarkets', 'Studying/working late'),
 (103021, 120051, '2-3 times a week', 'Before exercise', 'Increased energy and focus', 'Yes', 'Negative', 'Healthy', 'No', 3, 'Not available locally', 'Gangster', 'Brand reputation', 'More natural ingredients', 'Caffeine', 'No', 'Yes', 'Online ads', 'Innovative bottle design', 'No', '50-99', 'Online retailers', 'Social outings/parties'),
 (103024, 120054, 'Daily', 'Before exercise', 'Increased energy and focus', 'Yes', 'Positive', 'Not sure', 'Yes', 1, 'Health concerns', 'Bepsi', 'Taste/flavor preference', 'Healthier alternatives', 'Sugar', 'No', 'Yes', 'Online ads', 'Innovative bottle design', 'No', '100-150', 'Online retailers', 'Sports/exercise'),
 (103025, 120055, 'Once a week', 'Before exercise', 'To enhance sports performance', 'Yes', 'Neutral', 'Effective', 'Yes', 3, 'Not available locally', 'Cola-Coka', 'Availability', 'Healthier alternatives', 'Sugar', 'No', 'No', 'Outdoor billboards', 'Eco-friendly design', 'No', 'Above 150', 'Gyms and fitness centers', 'Studying/working late'),
 (103028, 120058, 'Once a week', 'Throughout the day', 'To boost performance', 'Yes', 'Positive', 'Not sure', 'No', 5, 'Other', 'Sky 9', 'Effectiveness', 'Reduced sugar content', 'Caffeine', 'No', 'Not Sure', 'TV commercials', 'Compact and portable cans', 'Yes', '100-150', 'Supermarkets', 'Studying/working late')]

In [7]:
%%sql
select * from dim_respondents limit 10;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
10 rows affected.


[(120031, 'Aniruddh Issac', '15-18', 'Female', 'CT117'),
 (120032, 'Trisha Rout', '19-30', 'Male', 'CT118'),
 (120033, 'Yuvraj  Virk', '15-18', 'Male', 'CT116'),
 (120034, 'Pranay Chand', '31-45', 'Female', 'CT113'),
 (120035, 'Mohanlal Joshi', '19-30', 'Female', 'CT120'),
 (120036, 'Zeeshan Ratta', '19-30', 'Female', 'CT118'),
 (120037, 'Oorja Anne', '19-30', 'Male', 'CT112'),
 (120038, 'Rhea Khanna', '19-30', 'Male', 'CT116'),
 (120039, 'Zara Joshi', '46-65', 'Male', 'CT116'),
 (120040, 'Sana Dhawan', '19-30', 'Female', 'CT116')]

In [8]:
%%sql
SELECT * from dim_cities;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
10 rows affected.


[('CT111', 'Delhi', 'Tier 1'),
 ('CT112', 'Mumbai', 'Tier 1'),
 ('CT113', 'Bangalore', 'Tier 1'),
 ('CT114', 'Chennai', 'Tier 1'),
 ('CT115', 'Kolkata', 'Tier 2'),
 ('CT116', 'Hyderabad', 'Tier 1'),
 ('CT117', 'Ahmedabad', 'Tier 2'),
 ('CT118', 'Pune', 'Tier 2'),
 ('CT119', 'Jaipur', 'Tier 2'),
 ('CT120', 'Lucknow', 'Tier 2')]

There are some responses where the respondent has not heard of the brand before but has responded as 'Yes' to the question of whether they have tried before. This is a contradiction. Hence we set the tried_before column to 'No' whenever the response to heard_before column is 'No'

In [9]:
%%sql
update fact_survey_responses
set tried_before = 'No'
where heard_before = 'No'

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5553 rows affected.


[]

# 1. Demographic insights  

##### Q1. Who prefers energy drink more? (male/female/non-binary?)

In [10]:
%%sql

select gender, count(gender) as no_of_respondents
from dim_respondents a
join fact_survey_responses b
on a.respondent_id = b.respondent_id
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
3 rows affected.


[('Male', 6038), ('Female', 3455), ('Non-binary', 507)]

It indicates that majority of the population who prefer energy drinks is consists of **Males**, followed by **females**

##### Q2. Which age group prefers energy drinks more?

In [11]:
%%sql

select age, count(age) as no_of_respondents
from dim_respondents a
join fact_survey_responses b
on a.respondent_id = b.respondent_id
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('19-30', 5520),
 ('31-45', 2376),
 ('15-18', 1488),
 ('46-65', 426),
 ('65+', 190)]

It is found that majority (5520) of the 10000 respondents who prefer energy drink belong to the '19-30' age group of young adults followed by older ones in the age group of '31-45'

##### Q3. Which type of marketing reaches the most Youth (15-30)

In [12]:
%%sql
select marketing_channels, count(marketing_channels) as no_of_respondents
from dim_respondents a
join fact_survey_responses b
on a.respondent_id = b.respondent_id
where age in ('15-18','19-30')
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Online ads', 3373),
 ('TV commercials', 1785),
 ('Other', 702),
 ('Outdoor billboards', 702),
 ('Print media', 446)]

It is thus clear that the **Online ads** are the primary marketing channels which reach the youth belonging to the age group **15-30**

# 2. Consumer Preferences
##### Q1. What are the preferred ingredients of energy drinks among respondents

In [13]:
%%sql

select ingredients_expected, count(ingredients_expected) as no_of_respondents
from fact_survey_responses
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
4 rows affected.


[('Caffeine', 3896), ('Vitamins', 2534), ('Sugar', 2017), ('Guarana', 1553)]

**Caffeine** is the ingredient that is most preferred followed by **vitamins**

##### Q2. What packaging preferences do respondents have for energy drinks?

In [14]:
%%sql

select packaging_preference, count(packaging_preference) as no_of_respondents
from fact_survey_responses
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Compact and portable cans', 3984),
 ('Innovative bottle design', 3047),
 ('Collectible packaging', 1501),
 ('Eco-friendly design', 983),
 ('Other', 485)]

Most people would prefer a packaging that is **compact and portable**. However a similar amount of people would also prefer **Innovative bottle designs**

# 3. Competition Analysis
##### Q1. Who are the current market leaders?

In [15]:
%%sql

select current_brands, count(1) as no_of_respondents
from fact_survey_responses
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
7 rows affected.


[('Cola-Coka', 2538),
 ('Bepsi', 2112),
 ('Gangster', 1854),
 ('Blue Bull', 1058),
 ('CodeX', 980),
 ('Sky 9', 979),
 ('Others', 479)]

**Cola-Coka** is the market leader with 2538 preffering it followed by **Bepsi** as a close second with 2112 respondents preferring it. **Gangster** is also one of the top market leaders preffered by 1854 respondents.

##### Q2. What are the primary reasons consumers prefer those brands over ours?

In [16]:
%%sql

select reasons_for_choosing_brands, count(1)  as no_of_respondents
from fact_survey_responses
where current_brands != 'CodeX'
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Brand reputation', 2393),
 ('Taste/flavor preference', 1829),
 ('Availability', 1715),
 ('Effectiveness', 1572),
 ('Other', 1511)]

Primary reason by brands

In [17]:
%%sql
WITH ranked_counts AS (
  SELECT current_brands, reasons_for_choosing_brands, COUNT(2) AS count,
    ROW_NUMBER() OVER (PARTITION BY current_brands ORDER BY COUNT(2) DESC) AS rank
  FROM fact_survey_responses
  WHERE current_brands != 'CodeX'
  GROUP BY 1, 2
)
SELECT current_brands, reasons_for_choosing_brands, count as no_of_respondents
FROM ranked_counts
WHERE rank = 1
ORDER BY current_brands;


 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
6 rows affected.


[('Bepsi', 'Brand reputation', 577),
 ('Blue Bull', 'Brand reputation', 289),
 ('Cola-Coka', 'Brand reputation', 616),
 ('Gangster', 'Brand reputation', 511),
 ('Others', 'Brand reputation', 140),
 ('Sky 9', 'Brand reputation', 260)]

Primary reason apart from Brand reputation by brands

In [18]:
%%sql
WITH ranked_counts AS (
  SELECT current_brands, reasons_for_choosing_brands, COUNT(2) AS count,
    ROW_NUMBER() OVER (PARTITION BY current_brands ORDER BY COUNT(2) DESC) AS rank
  FROM fact_survey_responses
  WHERE current_brands != 'CodeX'
  GROUP BY 1, 2
)
SELECT current_brands, reasons_for_choosing_brands, count as no_of_respondents
FROM ranked_counts
WHERE rank = 2
ORDER BY current_brands;


 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
6 rows affected.


[('Bepsi', 'Taste/flavor preference', 423),
 ('Blue Bull', 'Taste/flavor preference', 237),
 ('Cola-Coka', 'Taste/flavor preference', 531),
 ('Gangster', 'Taste/flavor preference', 357),
 ('Others', 'Effectiveness', 87),
 ('Sky 9', 'Taste/flavor preference', 194)]

Thus, considering overall reasons for choosing other brands over ours as well primary and secondary reasons by individual brands, it is clear that the respondents prefer other brands mainly due to their **Brand reputation** and **Taste/flavor experience**

# 4. Marketing Channels and Brand Awareness
##### Q1. Which marketing channel can be used to reach more customers?  
##### Q2. How effective are different marketing strategies and channels in reaching our customers?

In [19]:
%%sql
select marketing_channels, count(marketing_channels) as no_of_respondents
from fact_survey_responses 
where current_brands = 'CodeX'
group by 1 
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Online ads', 411),
 ('TV commercials', 261),
 ('Outdoor billboards', 119),
 ('Other', 116),
 ('Print media', 73)]

Our customers primarily come across energy drink ads Online followed by TV commercials. Thus promoting the use of Online ads alongside TV commercials can help us reach more customers.

# 5. Brand Penetration
##### Q1. What do people think about our brand? (overall rating)

In [20]:
%%sql

select round(avg(taste_experience),2) as rating
from fact_survey_responses
where current_brands = 'CodeX';

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
1 rows affected.


[(Decimal('3.27'),)]

##### Q2. Which cities do we need to focus more on?

In [21]:
%%sql

select c.city, count(a.respondent_id) as no_of_respondents
from fact_survey_responses a
join dim_respondents b on a.respondent_id = b.respondent_id
join dim_cities c
on b.city_id = c.city_id
where current_brands = 'CodeX'
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
10 rows affected.


[('Bangalore', 292),
 ('Hyderabad', 182),
 ('Mumbai', 156),
 ('Pune', 92),
 ('Chennai', 92),
 ('Kolkata', 48),
 ('Ahmedabad', 45),
 ('Delhi', 40),
 ('Jaipur', 28),
 ('Lucknow', 5)]

We have a smaller target audience in cities such as **Kolkata**, **Ahmedabad**, **Delhi**, **Jaipur** and **Lucknow**. Thus we can choose to focus more on these cities as part of a penetration strategy.  

Conversely, we already have a larger target audience in cities such as **Banglore**, **Hyderabad**, **Mumbai**, **Pune** and **Chennai**. Thus, we can opt to focus more on these cities as part of a market saturation strategy. 

# 6. Purchase Behavior

##### Q1. Where do respondents prefer to purchase energy drinks?

In [22]:
%%sql
select 	purchase_location, count(purchase_location) as no_of_respondents
from fact_survey_responses 
group by 1 
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Supermarkets', 4494),
 ('Online retailers', 2550),
 ('Gyms and fitness centers', 1464),
 ('Local stores', 813),
 ('Other', 679)]

4494 respondents prefer to purchase drinks at Supermarkets. However, a significant number (2550) of respondents also prefer purchasing energy drinks from Online Retailers. They are also available at many Gyms and Fitness centers as many athletes do consume them.

##### Q2. What are the typical consumption situations for energy drinks among respondents?

In [23]:
%%sql
select typical_consumption_situations, count(1) as no_of_respondents
from fact_survey_responses
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Sports/exercise', 4494),
 ('Studying/working late', 3231),
 ('Social outings/parties', 1487),
 ('Other', 491),
 ('Driving/commuting', 297)]

Majority of respondents consume energy drinks during **Sports/ Exercise**. However, people who **study** or **work** for **longer durations** also use energy drinks more than others.

##### Q3. What factors influence respondents' purchase decisions, such as price range and limited edition packaging?

In [24]:
%%sql
select limited_edition_packaging, count(1) as no_of_respondents
from fact_survey_responses
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
3 rows affected.


[('No', 4023), ('Yes', 3946), ('Not Sure', 2031)]

In [25]:
%%sql
select price_range, count(1) as no_of_respondents
from fact_survey_responses
group by 1
order by 2 desc;


 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
4 rows affected.


[('50-99', 4288), ('100-150', 3142), ('Above 150', 1561), ('Below 50', 1009)]

4023 respondents say that limited edition packaging won't affect their decision to buy a drink while a similar amount (3946) of respondents believe that it may affect their decision.  

A wide majority of respondents consider the price range of **50-99** to be reasonable while a significant amount (3142) of respondents prefer **100-150** price range.

# 7. Product Development
##### Q. Which area of business should we focus more on our product development? (Branding/taste/availability)

In [26]:
%%sql
select reasons_for_choosing_brands, count(1) as no_of_respondents
from fact_survey_responses
where current_brands = 'CodeX'
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Brand reputation', 259),
 ('Availability', 195),
 ('Taste/flavor preference', 182),
 ('Effectiveness', 176),
 ('Other', 168)]

In [27]:
%%sql
select tried_before, count(1)
from fact_survey_responses
group by 1

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
2 rows affected.


[('No', 7974), ('Yes', 2026)]

In [28]:
%%sql
select reasons_preventing_trying, count(1) as no_of_respondents
from fact_survey_responses
where tried_before = 'No'
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Not available locally', 1915),
 ('Health concerns', 1807),
 ('Not interested in energy drinks', 1788),
 ('Unfamiliar with the brand', 1483),
 ('Other', 981)]


Out of the 7974 respondents who haven't tried our energy drink before, 3595 respondents havent tried due to health concerns or no interest in energy drinks, however 1915 respondents were unable to try the drink as it was not available locally.  
Also, 1486 respondents were unfamiliar with the brand.  
So, focusing on availability should take top priority
followed by branding.

# Other interesting insights

In [29]:
%%sql

with cte as(
select sum(case when heard_before ='Yes' then 1 else 0 end) as yes,
sum(case when heard_before='No' then 1 else 0 end) as no
from fact_survey_responses)
select round(((cast(yes as float)/(yes+no))*100)::numeric,2) as percent_heard_before
from cte;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
1 rows affected.


[(Decimal('44.47'),)]

Out of the 10000 respondents, only **44.47%** of respondents had heard of our brand.

In [30]:
%%sql
with cte as(
select sum(case when tried_before ='Yes' then 1 else 0 end) as yes,
sum(case when tried_before='No' then 1 else 0 end) as no
from fact_survey_responses)
select round(((cast(yes as float)/(yes+no))*100)::numeric,2) as percent_tried_before
from cte;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
1 rows affected.


[(Decimal('20.26'),)]

Only **20.26%** of respondents had tried our brand before.

In [31]:
%%sql
with cte as(
select sum(case when heard_before = 'Yes' and tried_before ='Yes' then 1 else 0 end) as yes_yes,
sum(case when heard_before = 'Yes' and tried_before='No' then 1 else 0 end) as yes_no,
sum(case when heard_before='Yes' then 1 else 0 end) as yes
from fact_survey_responses)
select round(((cast(yes_yes as float)/yes)*100)::numeric,2) as heard_and_tried, 
round(((cast(yes_no as float)/yes)*100)::numeric,2) as heard_but_not_tried
from cte;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
1 rows affected.


[(Decimal('45.56'), Decimal('54.44'))]

And, **45.45%** of respondents had heard of our energy drink before and also tried it, which means **54.44%** of respondents hadn't tried our energy drink despite hearing about it.

In [32]:
%%sql
select improvements_desired, count(improvements_desired) as no_of_respondents
from fact_survey_responses
where current_brands='CodeX'
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
5 rows affected.


[('Reduced sugar content', 298),
 ('More natural ingredients', 234),
 ('Wider range of flavors', 208),
 ('Healthier alternatives', 143),
 ('Other', 97)]

Around 300 respondents think that our energy drink should have reduced sugar content while 234 respondents think that it should contain more natural ingredients. A significant amount of respondents also believe that we should introduce a wider range of flavors.

In [33]:
%%sql
select ingredients_expected, count(1) as no_of_respondents
from fact_survey_responses
where current_brands='CodeX'
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
4 rows affected.


[('Caffeine', 363), ('Vitamins', 243), ('Sugar', 219), ('Guarana', 155)]

More than **350 respondents** who prefer our brand believe that our energy drink should primarily contain **Caffeine** while around **250 respondents believe** it should contain **Vitamins**. 

In [34]:
%%sql
select interest_in_natural_or_organic, count(1)
from fact_survey_responses
where current_brands='CodeX'
group by 1
order by 2 desc;

 * postgresql://postgres:***@localhost/Codebasics_Challenge_Project
3 rows affected.


[('Yes', 506), ('No', 273), ('Not Sure', 201)]

Almost half of our respondents would be interested in natural/ organic energy drinks.

#### NOTE: As part of this challenge, we were required to create a presentation to present the insights gained through the data analysis process. 

Thus, the data was visualized using **Tableau** and the visualizations were used in the presentation attached alongside the file.

# RECOMMENDATIONS for CodeX

##### **What immediate improvements can we bring to the product?**

**Reduced Sugar Content:** With majority of individuals desiring reduced sugar content, it is important for CodeX to focus on developing a formulation with lower sugar levels. This can involve exploring alternative sweeteners or reducing the overall sugar content while maintaining taste and consumer satisfaction. By offering a reduced sugar option, CodeX can cater to the increasing demand for healthier beverage choices.

**More Natural Ingredients:** CodeX can enhance its product by incorporating a greater variety of natural and organic ingredients. This can involve using natural flavorings, natural sweeteners, and avoiding artificial additives. By providing a product with recognizable and desirable ingredients, CodeX can cater to consumers seeking a cleaner and more natural energy drink option.

**Wider Range of Flavors:** CodeX can introduce new flavor options to its energy drink lineup. By offering a diverse range of flavors such as fruit-infused variations or unique blends, CodeX can appeal to a broader consumer base and cater to different taste preferences.  





##### **What should be the ideal price of our product?**

● Considering the high number of respondents (4288) who find the $50-99 range reasonable, it would be wise to position the ideal price of CodeX energy drinks within this range. This price point aligns with the preferences of the majority and provides an attractive and competitive price range for potential customers.

● However, it's also worth considering introducing product variants or limited editions at slightly higher price points to cater to the preferences of the remaining segments. This strategy allows CodeX to capture the market segments that prefer the $100-150 and above $150 price ranges, thus maximizing revenue opportunities.

● It's essential to conduct a comprehensive pricing analysis, taking into account factors such as production costs, profit margins, competitor pricing, and perceived value. This analysis will help determine the ideal price point within the $50-99 range while considering potential premium offerings for higher price segments.

● Regularly monitor market dynamics, competitor pricing, and consumer feedback to make necessary adjustments and ensure CodeX remains competitive in the energy drink market.


##### **What kind of marketing campaigns, offers, and discounts we can run?**  

**Influencer Partnerships:** Collaborate with popular athletes, fitness influencers, and social media personalities who align with the brand's target audience. Have them promote CodeX energy drinks through sponsored content, engaging posts, and testimonials to reach a wider audience.

**Social Media Giveaways:** Conduct regular giveaways on social media platforms to generate buzz and increase brand engagement. Encourage followers to share their energetic moments or creative workout routines using a specific hashtag, and randomly select winners to receive CodeX merchandise or exclusive discounts.

**Event Sponsorships:** Sponsor sports events, fitness competitions, or health expos where the target audience is likely to be present. Set up booths or sampling stations to offer attendees the opportunity to try CodeX energy drinks, distribute branded merchandise, and provide event-exclusive discounts.

**Partnerships with Fitness Centers and Gyms:** Collaborate with local fitness centers and gyms to cross-promote CodeX energy drinks. Offer special discounts or branded promotions to their members, and place CodeX products in on-site vending machines or smoothie bars.

**Seasonal Promotions:** Create seasonal promotions and limited-time offers to create a sense of urgency and excitement. For example, introduce special edition flavors or offer bundled packs of CodeX energy drinks with related merchandise during holidays or peak fitness seasons.


##### **Who can be a brand ambassador, and why?**  

**Professional Athletes:** Consider partnering with professional athletes who are known for their dedication to fitness and have a strong influence on the target audience. These athletes can endorse CodeX energy drinks as a source of energy and performance enhancement during workouts and competitions. Their association with the brand can enhance credibility and attract fitness enthusiasts.

**Fitness Influencers:** Collaborate with popular fitness influencers who have a large following on social media platforms, particularly those who align with the target audience's interests and values. Fitness influencers can showcase how CodeX energy drinks contribute to their active lifestyles, sharing personal experiences, and promoting the product's benefits to their engaged followers.

**Local Fitness Instructors:** Engage with local fitness instructors, trainers, and coaches who have a strong influence within their communities. These individuals can advocate for CodeX energy drinks during their fitness classes and training sessions, showcasing the product as a fueling and refreshing option for their clients.

##### **Who should be our target audience, and why?**  

● The primary target audience for CodeX should be young adults between the ages of 19 and 30. This age group demonstrates the highest preference for energy drinks, closely followed by the age group of 31-45 and 15-18.

● Furthermore, considering the typical consumption situations for energy drinks among respondents, such as sports/exercise and studying/working late, the target audience should consist of individuals engaged in active lifestyles, including athletes, fitness enthusiasts, and students/professionals with demanding schedules.

● In terms of gender, the preference for energy drinks is higher among males, followed by females and non-binary individuals. Therefore, the target audience should cater to all genders but with a particular focus on males.

● Additionally, since online ads and TV commercials are the most effective marketing channels for reaching the youth (15-30), CodeX should prioritize digital marketing campaigns and engaging advertisements on television to effectively reach and engage with the target audience.

● In summary, the recommended target audience for CodeX energy drinks should be young adults aged 19-30, with a focus on individuals engaged in sports, exercise, studying, or working late. The marketing efforts should prioritize online platforms and TV commercials to effectively reach and resonate with this target audience.